src/nft/components/bag/BagRow.tsx

```diff
@@ -19,9 +20,18 @@ import { GenieAsset, UpdatedGenieAsset } from 'nft/types'
 import { ethNumberStandardFormatter, formatWeiToDecimal, getAssetHref } from 'nft/utils'
 import { MouseEvent, useCallback, useEffect, useReducer, useState } from 'react'
 import { Link } from 'react-router-dom'
+import styled from 'styled-components/macro'
 
 import * as styles from './BagRow.css'
 
+const RemoveButton = styled(MediumButton)`
+  border-radius: 12px;
+  font-size: 14px;
+  line-height: 16px;
```

Comment:
Do our styled components have shared typography styles? NFT has some in `common.css.ts` which would allow for these two to be pulled from a central style for `buttonTextSmall`

Reply:
we have the core ones in `src/theme/index#ThemedText`, but I don't think it has been updated to support all our current text styles (there was a large update recently)